src/civic/index.js

```diff
@@ -17,31 +18,31 @@ const { logger } = require('../logging');
 const _pubmed = require('../entrez/pubmed');
 const _entrezGene = require('../entrez/gene');
 const { civic: SOURCE_DEFN, ncit: NCIT_SOURCE_DEFN } = require('../sources');
-const { downloadVariantRecords, processVariantRecord } = require('./variant');
+const { processVariantRecord } = require('./variant');
 const { getPublication } = require('./publication');
-const { evidence: evidenceSpec } = require('./specs.json');
+const { EvidenceItem: evidenceSpec } = require('./specs.json');
 
 class NotImplementedError extends ErrorMixin {}
 
 const ajv = new Ajv();
 
-const BASE_URL = 'https://civicdb.org/api';
+const BASE_URL = 'https://civicdb.org/api/graphql';
 
 /**
- * https://civicdb.org/glossary
+ * https://docs.civicdb.org/en/latest/model/evidence/level.html
```

Comment:
I think this should be:
`https://docs.civicdb.org/en/latest/model/evidence/evidence_rating.html`

Reply:
There is https://docs.civicdb.org/en/latest/model/evidence/level.html for the evidence Level and https://docs.civicdb.org/en/latest/model/evidence/evidence_rating.html for the Evidence Rating. Do we want both? And what about the url at line 45?

Reply:
I suppose we should have both because we're defining the ratings (1-5) and the levels (A-E), or we could use a more general one for the evidence item page where the other two links can be found:
https://docs.civicdb.org/en/latest/model/evidence.html

Reply:
In the previous version I concatenated them

Reply:
ok. I propose to put them both in the comment section and use https://docs.civicdb.org/en/latest/model/evidence.html for the url field.